### ISSUE TITLE
DAVE BYRON: Split the data on Google sheets to reflect NT / OT/ OBS

### ISSUE THREAD

**Issue by Redmayspecial** [unfoldingWord-dev/translationDatabaseWeb#413]
If possible to the data in the format that it needs to be.  Fields are only allowed to have specific values.

**Comment by vleong2332** [comment]
tQ and tN are currently split into tQ/N OBS and tQ/N Bible. According to @benjore, they should now be split into OBS, OT, and NT. Confirm with Dave Byron that's they way we want to go. If positive, we need the spreadsheet be updated to reflect those changes.

**Comment by vleong2332** [comment]
Blocked. Waiting for update spreadsheet from Dave Byron's team.

**Comment by Redmayspecial** [comment]
Mark Stedman:
I propose we “go-live” on using the Gateway Language Tracker on tD as the authoritative source for the status of GL projects as soon as possible. It feels like this transition is dragging along because it doesn’t have a defined end-point.
 
The outstanding issue at this point is for the GL Team to adjust the data on the current Tracker Spreadsheet to correctly reflect how you view the Phases. I understand that the GL Team is moving some items around and creating/removing others. This action item rests with Translation Services.
 
Once that work is complete, we will import 100% of that data into the GL Tracker and be ready to go.
 
I propose that we Go-Live on 3/10. In order for our team to hit that goal we’ll need the data on the GL Tracker Google Doc to be corrected by 3/3.

**Comment by Redmayspecial** [comment]
Dave Byron:
Am I understanding you correctly that I:
1). Need to finalize any changes to the GL Tracker doc that show the ULB, the UDB, and the Notes split in half to reflect the new 30 Sept goals,
2). Make sure the GLD's have those new columns correctly populated,
3). All of this done before 3 March
4).  And no further changes to the GLTracker made after 3March?

**Comment by vleong2332** [comment]
We have enough info to proceed. Superseded by #479
